py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -80,23 +81,45 @@ def index(
     source_ir: Optional[SourceIR],
     name: str,
     input: TRTTensor,
-    index: Union[TRTTensor, Sequence[TRTTensor]],
+    index: Union[
+        TRTTensor,
+        Sequence[TRTTensor],
+        np.ndarray,
+        Sequence[np.ndarray],
+        torch.Tensor,
+        Sequence[torch.Tensor],
+    ],
```

Comment:
This can be shortened to `index: Sequence[Union[TRTTensor, np.ndarray, torch.Tensor]]`, since the subsequent code assumes `index` is a sequence of tensors.